keyboards/pizzakeyboards/slice65/info.json

```diff
@@ -0,0 +1,100 @@
+{
+    "manufacturer": "Pizzakeyboards",
+    "keyboard_name": "Slice65",
+    "maintainer": "mm0nte",
+    "board": "GENERIC_STM32_F303XC",
+    "bootloader": "stm32-dfu",
+    "debounce": 10,
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "nkro": true
+    },
```

Comment:
```suggestion
    },
    "qmk": {
        "locking": {
            "enabled": true,
            "resync": true
        }
    },
```

Reply:
I see moved from config.h...

Reply:
FYI: After I added this and ran `qmk format-json -i info.json`, it removed the comma after `"nkro": true`.